awssns/main.go

```diff
@@ -215,7 +215,7 @@ func (clients Clients) HandleNotification(w http.ResponseWriter, r *http.Request
 
 		event := cloudevents.Event{
 			Context: cloudevents.EventContextV03{
-				Type:            data["Type"].(string),
+				Type:            "com.amazon.sns",
```

Comment:
same comment here. Isn't SNS more like a transport than something that describes the event?